### ISSUE TITLE
no-invalid-position-declaration false negative (@include @layer)

### ISSUE THREAD

**Issue by fbasmaison-lucca** [stylelint-scss/stylelint-config-standard-scss#269]
Using the following code…

```scss
@mixin root {
	:root {
		@content;
	}
}

@include root {
	@layer my-layer {
		--custom-prop: 'value';
	}
}
```

… results in the following stylelint error: _Unexpected invalid position declaration_.

The resulting CSS is valid:

```css
@layer my-layer {
  :root {
    --custom-prop: "value";
  }
}
```

<img width="988" height="221" alt="Image" src="https://github.com/user-attachments/assets/e7d3e545-9fe2-4026-bf13-618a8f43c503" />
